### PR TITLE
The Powder Toy Game

### PR DESCRIPTION
**Pull request by Muvkina**
I quite enjoyed this game I also changed the readme because I think sandbox games should have their own section but I can remove this.

### CODE REVIEW

File: games/thepowdertoy/depends [hunk 1]
```diff
@@ -0,0 +1,2 @@
+scons
+sdl2
```
Comment (sdsddsd1) [inline games/thepowdertoy/depends]: Pls remove the trailing white space. Also ´scons´ maybe make dep?

**Comment by sdsddsd1** [comment]
We can add a new section. I kept the sections small and only extended when needed.

**Comment by Muvkina** [comment]
fyi the sed I don't believe affects xorg builds although I haven't tested it. But it fixes wayland. Perhaps if it does affect xorg have a switch based off of kiss l wayland?

**Comment by Muvkina** [comment]
I think that fixed the whitespace I didn't see any in vim

**Comment by sdsddsd1** [comment]
I have no KISS computer at the moment to test, but I would fiddle something like

```
kiss l wayland && sed -i '/X11/d' SConscript
```

**Comment by Muvkina** [comment]
Fixed and squashed into one commit

**Comment by sdsddsd1** [comment]
Does `kiss-outdated` return something for the current pkg name?
Going after repology it _should_ be `powdertoy`

**Comment by Muvkina** [comment]
my bad can fix that and resquash

**Comment by Muvkina** [comment]
Fixed

**Comment by sdsddsd1** [comment]
Btw, this game looks awesome :D

**Comment by Muvkina** [comment]
Yeah I've been really into sand simulations lately and found this on github there's some really cool creations built in it. I saw stuff like a computer and a typewriter.

ps: on irc do you usually go by claudia02?

**Comment by sdsddsd1** [comment]
yes.

**Pull request merged**